docs/usage/requests.rst

```diff
@@ -138,3 +138,25 @@ for ``Body`` , by using :class:`RequestEncodingType.MESSAGEPACK <.enums.RequestE
 .. literalinclude:: /examples/request_data/msgpack_request.py
    :caption: msgpack_request.py
    :language: python
+
+
+Custom Request
+--------------
+
+.. versionadded:: 2.7.0
```

Comment:
I agree on the consistency but I was also thinking that if we start it, then it would never be done or require a lot of digging into when things were added to go back and add them.

Im up in the air on this